cli/cli.go

```diff
@@ -73,7 +74,11 @@ func Init(version string, confPaths []string) {
 	// Linter:
 	lintCmd := app.Command("lint", "Runs a linter on Tyk configuration file")
 	lintCmd.Action(func(c *kingpin.ParseContext) error {
-		path, lines, err := lint.Run(lint.ConfSchema, confPaths)
+		confSchema, err := ioutil.ReadFile("cli/linter/schema.json")
```

Comment:
Should we use a more specific path here? Just thinking about potential issues

Reply:
The working directory is root. I think we should say where the json is.